joern-cli/src/main/scala/io/shiftleft/joern/JoernExport.scala

```diff
@@ -3,33 +3,91 @@ package io.shiftleft.joern
 import better.files.File
 import better.files.Dsl._
 import io.shiftleft.codepropertygraph.Cpg
-import io.shiftleft.dataflowengineoss.layers.dataflows.{Cpg14DumpOptions, DumpCpg14, OssDataFlow, OssDataFlowOptions}
+import io.shiftleft.dataflowengineoss.layers.dataflows.{
+  Cpg14DumpOptions,
+  DdgDumpOptions,
+  DumpCpg14,
+  DumpDdg,
+  DumpPdg,
+  OssDataFlow,
+  OssDataFlowOptions,
+  PdgDumpOptions
+}
 import io.shiftleft.dataflowengineoss.semanticsloader.Semantics
 import io.shiftleft.joern.console.JoernWorkspaceLoader
 import io.shiftleft.semanticcpg.language._
-import io.shiftleft.semanticcpg.layers.LayerCreatorContext
+import io.shiftleft.semanticcpg.layers.{
+  AstDumpOptions,
+  CdgDumpOptions,
+  CfgDumpOptions,
+  DumpAst,
+  DumpCdg,
+  DumpCfg,
+  LayerCreatorContext
+}
 
-case class ExporterConfig(cpgFileName: String = "cpg.bin", outDir: String = "out")
+case class ExporterConfig(cpgFileName: String = "cpg.bin", outDir: String = "out", repr: String = "cpg14")
```

Comment:
cpg14?

Reply:
ah ok i guess i got it - cpg2014. maybe worth documenting at some point :)

Reply:
Yep. Documentation will follow in a separate PR.